docs/algorithms/sis-lattice.rst

```diff
@@ -0,0 +1,36 @@
+.. _SIS Lattice Attacks:
+
+SIS Lattice Attacks
+=====================
+
+We construct an (easy) example SIS instance::
+
+    from estimator import *
+    params = SIS.Parameters(n=113, q=2048, length_bound=512, norm=2) 
+    params
+
+The simplest (and quickest to estimate) model is solving for the SIS instance with a euclidian norm length bound and assuming the Gaussian heuristic [CheNgu12]_.Then, we can solve for the required root hermite factor [EC:GamNgu08]_ that will guarantee BKZ outputs a short enough vector::
```

Comment:
Why are we referencing CheNgu12 here?

Reply:
Fair enough. Couldn't exactly find a better reference for the Gaussian heuristic. Removed.

Reply:
Ah, sorry, I misread, yes, this is fine. Sorry for the noise